aidial_assistant/json_stream/json_array.py

```diff
@@ -28,50 +29,40 @@ def type(self) -> str:
     def token() -> str:
```

Comment:
Comment for `json_bool.TRUE_STRING` and `json_bool.FALSE_STRING` - they are not unused anywhere.